File: src/test/java/com/mmg/StudentTest.java
package com.mmg;

import com.mmg.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void hightestTest(){
        Student student = new Student("MMG",80,90);
        Assertions.assertEquals(90,student.hightest());
    }
    @Test
    public void averageTest(){
        Student student = new Student("MMG",80,90);
        Assertions.assertEquals((80+90)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/mmg/StudentTest.kt
package com.mmg.kotlin

import com.mmg.student.kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest(){
    @Test
    fun gradingTest(){
        val student = Student("MMG",100,100)
        Assertions.assertEquals('A',student.grading())
    }
    @Test
    fun averageTest(){
        val student = Student("MMG",100,100)
        Assertions.assertEquals(100,student.getAverage())
    }
    @Test
    fun passTest(){
        val student = Student("MMG",100,100)
        Assertions.assertEquals("\tPASS",student.passOrFailed())
    }
}

